eng/config/PublishData.json

```diff
@@ -173,7 +173,8 @@
       "packageFeeds": "default",
       "channels": [],
       "vsBranch": "rel/d16.10",
-      "vsMajorVersion": 16
+      "vsMajorVersion": 16,
+      "insertionTitlePrefix": "[d16.10]"
```

Comment:
I have found these kinds of title prefixes more useful than "Auto Insertion", thanks @Cosifne for being the first to do it :)